index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portofolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <div class="wrapper">
+
+        <nav class="flex-Nav">
+            <h1 class="header-Name">Zeeshan Haider</h1>
+            <button class="nav-Button"><img src="images/menu.png" alt="Menu"></button>
+        </nav>
```

Comment:
- [ ] To keep the HTML semantic, I would suggest some changes to this section. The whole section should be wrapped in a `<header>` instead of the `<nav>.
- [ ] Your name, which is the logo here, should be in an `<a>` tag, instead of `<h1>`. Usually, the logo links to the main page.
> Keep in mind that even though using more than one `<h1>` is allowed by the HTML specification, is not considered a good practice, you can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).
- [ ] To keep it semantic, you should consider wrapping menu image in a `<nav>` tag, you can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav).

Reply:
I've put the menu in a Nav and wrapped the section in header now.